documentation/manpages/tool/update-man-pages.sh

```diff
@@ -23,11 +30,29 @@ else
   exit 1
 fi
 
-ls docs-main/docs/core/tools/dotnet*.md | while read -r line;
+ls docs-main/docs/core/tools/dotnet*.md | while read -r file;
   do
-    echo "Working on $line"
-    "$MANPAGE_TOOL_DIR"/remove-metadata-and-embed-includes.py "$line"
-    pandoc -s -t man -V section=1 -V header=".NET" --column=500 --filter "$MANPAGE_TOOL_DIR"/man-pandoc-filter.py "$line" -o "$(basename "${line%.md}".1)"
+    echo "Working on $file"
+    "$MANPAGE_TOOL_DIR"/remove-metadata-and-embed-includes.py "$file"
+    command_name=$(basename "${file%.md}")
+    man_page_section=1
+    if [[ "$command_name" == "dotnet-install-script" ]]; then
+        continue;
+    fi
+    if [[ "$command_name" == "dotnet-environment-variables" ]] || \
+       [[ "$command_name" == "dotnet-new-sdk-templates" ]]; then
+        "$MANPAGE_TOOL_DIR"/handle-missing-name.py "$file"
+        man_page_section=7
+    fi
+    file_modification_date=$(git -C docs-main/ log -1 --format='%cs' "${file#docs-main/}")
+    pandoc --standalone -t man \
+      -V title="$command_name" \
+      -V section="$man_page_section" \
+      -V header=".NET Documentation" \
+      -V date="$file_modification_date" \
+      --column=500 \
+      --filter "$MANPAGE_TOOL_DIR"/man-pandoc-filter.py \
+      "$file" -o "${command_name}"."${man_page_section}"
 done
 
-rm -rf docs-main
+# rm -rf docs-main
```

Comment:
I have found leaving the docs-main repo alone quite useful for iterating quickly. Otherwise, every time I run this script, it clones a fresh copy of dotnet/docs repo, which takes a long time. I can add back the `rm -rf` if it's still desirable.

Reply:
Any thoughts on this, @baronfel ?

Reply:
Perfectly happy to have it commented out for ease of use.